packages/fluent-editor/src/table-up/index.ts

```
@@ -0,0 +1,10 @@
+import { TableUp as ImportTableUp } from 'quill-table-up'
+
+export const OriginTableUp = ImportTableUp
+
+export class TableUp extends OriginTableUp {
+  static moduleName = 'table-up'
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)